### PR TITLE
media: Modify FileOutputDataSource return value

### PR DESCRIPTION
**Pull request by junmin-kim**
Description says `if error occurred, it returns -1`
Match description and implement

### CODE REVIEW

File: framework/src/media/FileOutputDataSource.cpp [hunk 1]
```diff
@@ -105,12 +105,12 @@ bool FileOutputDataSource::isPrepare()
 ssize_t FileOutputDataSource::write(unsigned char* buf, size_t size)
 {
 	if (!isPrepare()) {
-		return (ssize_t)0;
+		return (ssize_t)-1;
 	}
 
 	if (!buf) {
 		meddbg("buf is nullptr, hence return 0\n");
```
Comment (JeongJunSik) [inline framework/src/media/FileOutputDataSource.cpp]: hence return 0 ??

**Comment by seinfra** [comment]
Target : [468c25f203f0c2daa808f0ccbd8d8940ce30bd14] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [468c25f203f0c2daa808f0ccbd8d8940ce30bd14] - Code Rule Check (C++) OK.

File: framework/src/media/FileOutputDataSource.cpp (hunk 1 above)
Comment (junmin-kim) [reply framework/src/media/FileOutputDataSource.cpp]: return -1

**Comment by seinfra** [comment]
Target : [3a8020bce3107c0875b128f6b5f0843b58ef11ac] -

**Comment by seinfra** [comment]
Target : [3a8020bce3107c0875b128f6b5f0843b58ef11ac] -

**Comment by seinfra** [comment]
Target : [c0a0cb67d25ce22ab960b1b9ac6aec3f6dc74831] -

**Comment by seinfra** [comment]
Target : [c0a0cb67d25ce22ab960b1b9ac6aec3f6dc74831] -

**Comment by seinfra** [comment]
Target : [8a30258a376f96bfef961890f2a3735689dc0378] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [8a30258a376f96bfef961890f2a3735689dc0378] - Code Rule Check Result:
framework/include/media/FileOutputDataSource.h:42: ERROR: [SPC_M_OPR] spaces prohibited around that ':' (ctx:WxW)
framework/include/media/FileOutputDataSource.h:58: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FileOutputDataSource.h:58: ERROR: [SPC_M_OPR] need consistent spacing around '&' (ctx:VxW)
framework/include/media/FileOutputDataSource.h:68: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)
framework/include/media/FileOutputDataSource.h:68: ERROR: [SPC_M_OPR] need consistent spacing around '&' (ctx:VxW)
framework/include/media/FileOutputDataSource.h:74: ERROR: [SPC_M_OPR] need consistent spacing around '&' (ctx:VxW)
framework/include/media/FileOutputDataSource.h:80: ERROR: [SPC_M_OPR] need consistent spacing around '&' (ctx:VxW)
framework/include/media/FileOutputDataSource.h:80: ERROR: [SPC_M_OPR] spaces required around that '=' (ctx:VxV)
framework/include/media/FileOutputDataSource.h:80: ERROR: [SPC_M_OPR] need consistent spacing around '&' (ctx:VxW)
framework/include/media/FileOutputDataSource.h:122: ERROR: [SPC_M_OPR] spaces required around that ':' (ctx:OxV)

**Pull request merged**